README.md

```diff
@@ -0,0 +1,57 @@
+OpenSSL Technical Policies
+==========================
+
+This repository contains the technical policies and procedures established by
+the OTC in accordance with the [project bylaws] and the requirements specified
```

Comment:
OTC and OMC are first used here, OTC isn't defined until later (line 30).  What I've done else where is, on first reference, to have them [OTC] and [OMC] linking to the glossary entries.   Then the definition later isn't required -- keeping the definition in one place is good.

Reply:
Done.